packages/components/src/post-locked-modal/index.js

```diff
@@ -0,0 +1,120 @@
+/**
```

Comment:
It seems like this component would be better in the `editor` package as it seems that it's something dependent on the `core/editor` store and also that it's independent from `edit-post` because it can be reused in alternative Gutenberg editors (customization, standalone et...)

Reply:
Sure I can move it there, good point.

Reply:
Sure, I can take a look.